tests/phpunit/tests/KSES.php

```diff
@@ -308,6 +332,25 @@ public function test_filter_kses_allowed_html( $html, $expected ) {
 		remove_filter( 'wp_kses_allowed_html', [ $kses, 'filter_kses_allowed_html' ] );
 	}
 
+	/**
+	 * Testing the filter_kses_allowed_html() method.
+	 *
+	 * @covers ::filter_kses_allowed_html
+	 * @covers ::array_merge_recursive_distinct
+	 */
+	public function test_filter_kses_allowed_html_filter() {
+		$kses         = new \Google\Web_Stories\KSES();
+		$allowed_tags = [
+			'img' => [
+				'width' => true,
+			],
+		];
+
+		$result = $kses->filter_kses_allowed_html( $allowed_tags );
+		$this->assertArrayHasKey( 'img', $result );
+		$this->assertArrayHasKey( 'width', $result['img'] );
+	}
+
 	public function data_test_filter_kses_allowed_html() {
```

Comment:
Let's add a test case to the data provider here with a simple `img` tag

Reply:
Done